src/components/cms/ContentBuilder.tsx

```
@@ -0,0 +1,426 @@
+          <p>Éditeur pour {block.type} - En développement</p>
+        </div>
+      );
+  }
+};
+
+// Block Preview Component
+const BlockPreview: React.FC<{ block: ContentBlock }> = ({ block }) => {
+  switch (block.type) {
+    case 'heading':
+      const HeadingTag = `h${block.content.level}` as keyof JSX.IntrinsicElements;
+      return (
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)